packages/airnode-operation/src/evm/requests/make-requests.ts

```diff
@@ -72,7 +72,8 @@ export async function makeWithdrawal(state: State, request: Request) {
 
   const { AirnodeRrp } = state.contracts;
 
-  await AirnodeRrp.connect(signer).requestWithdrawal(airnode, sponsorWallet.address);
+  const tx = await AirnodeRrp.connect(signer).requestWithdrawal(airnode, sponsorWallet.address);
```

Comment:
This is part of the patch that @acenolaza shared to me which I've used to regenerate the fixtures. I didn't include it since it should be done more properly and its tracked in https://api3dao.atlassian.net/browse/AN-154. Gladly these fixtures do not change very often.

Reply:
I see you've been having fun with test fixtures 😄